genocrate/profile/genocrate-batch-submission/example/ro-crate-preview.html

```diff
@@ -0,0 +1,328 @@
+
+
+<html>
+<head>
+
```

Comment:
Is this HTML autogenerated? (in general would be the html be *not* included in the submissions - and then we'd generate a top-level ro-crates.json + HTML?)

Reply:
Yes, the HTML is auto-generated.